newrelic/core/stats_engine.py

```diff
@@ -1233,17 +1233,49 @@ def record_log_event(self, message, level=None, timestamp=None, attributes=None,
         timestamp = timestamp if timestamp is not None else time.time()
         level = str(level) if level is not None else "UNKNOWN"
 
-        if not message or message.isspace():
-            _logger.debug("record_log_event called where message was missing. No log event will be sent.")
+        # Unpack message and attributes from dict inputs
```

Comment:
I'm assuming this implementation is the same as the one above and the same feedback from the previous applies here as well.